pom.xml

```diff
@@ -90,14 +90,14 @@ under the License.
         <maven.version>2.2.1</maven.version>
         <maven-plugin-plugin.version>3.6.0</maven-plugin-plugin.version>
 
-        <scalac-scoverage-plugin.version>1.4.11</scalac-scoverage-plugin.version>
-        <scalac-scoverage-plugin.scala.version>2.12.15</scalac-scoverage-plugin.scala.version>
+        <scalac-scoverage-plugin.version>2.0.7</scalac-scoverage-plugin.version>

```

Comment:
should the version of this plugin also be bumped to correspond to the scalac-scoverage plugin? at least to 2.0.0 version?